src/components/ActionButton.tsx

```diff
@@ -37,14 +37,15 @@ export interface IActionButtonProps {
 export const ActionButton: React.FunctionComponent<IActionButtonProps> = (
   props: IActionButtonProps
 ) => {
+  const { disabled, className, title, onClick, icon } = props;
   return (
     <button
-      disabled={props.disabled}
-      className={classes(actionButtonStyle, props.className)}
-      title={props.title}
-      onClick={props.onClick}
+      disabled={disabled}
+      className={classes(actionButtonStyle, className)}
+      title={title}
+      onClick={onClick}
     >
-      <LabIcon.resolveReact icon={props.iconName} />
+      <LabIcon.resolveReact icon={icon} elementPosition="center" tag="span" />
```

Comment:
```suggestion
      <icon.react elementPosition="center" tag="span" />
```

Don't bother with `resolveReact` here. You already know/can enforce via `IActionButtonProps` that all `icon` here are instances of `LabIcon`.